documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -244,6 +244,15 @@ The `Customer OTP` rows are OTP registers 36 to 43 in the `vcgencmd otp_dump` ou
 
 Default: ""
 
+[.whitepaper, title="Changing MAC addresses", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003474-WP/Changing-MAC-addresses.pdf]
```

Comment:
This probably fits better at https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#industrial-use-of-the-raspberry-pi or https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#otp-register-and-bit-definitions ? Note that it also covers "devices up to the Pi 3" and so isn't specific to the BCM2711.

Reply:
Agreed. Although I'm not sure we should surface this one at all. Cut it.